Update the readme and documentation for guardian 1.0

hassox opened the pull request:
This should not be merged until Guardian 1.0 has shipped.

Commit:
Update the readme and documentation for guardian 1.0

doomspork commented on README.md:
```diff
@@ -2,7 +2,7 @@ GuardianDb
 
 GuardianDB is an extension to vanilla Guardian that tracks tokens in your
-application to prevent playback.
+applications database to prevent playback.
```
I think you want "application's" here.

doomspork commented on README.md:
````diff
@@ -42,100 +41,83 @@ You will then need to add a migration:
 ```
 
-**Do not run the migration yet.** You also need to add this to your configuration:
+**Do not run the migration yet.** You also need to finish your setup first.
````
This doesn't read quite right, what about something like:

**Do not run the migration yet**, we need to complete our setup first:

doomspork commented on README.md:
```diff
@@ -42,100 +41,83 @@ You will then need to add a migration:
+  config :guardian_db, GuardianDb,
+         repo: MyApp.Repo,
+         schema_name: "guardian_tokens", # default
+         sweep_interval: 60 # minutes. 60 default
```
You could combine these two fragments: `# default: 60 minutes`

doomspork commented on README.md:
```diff
@@ -42,100 +41,83 @@ You will then need to add a migration:
-After that, each time they are verified, the token is looked up. If present, the
-verification continues but if it is not found, the verification is abandoned
-with an error response.
+To sweep expired tokens from your db you should add `GuardianDb` to your supervision tree.
```
Did you mean `GuardianDb.ExpiredSweeper`?

doomspork commented on README.md:
````diff
@@ -42,100 +41,83 @@ You will then need to add a migration:
-      {:error, reason} -> # Oh no
-    end
-```
+GuardianDb works by hooking into the lifecycle of your token module.
````
In other places `GuardianDb` is in backticks, did you want to do that here too?

doomspork commented on README.md:
```diff
@@ -42,100 +41,83 @@ You will then need to add a migration:
+GuardianDb works by hooking into the lifecycle of your token module.
 
-It's a good idea to purge out any stale tokens that have already expired.
+You'll need to add it to
```
I think you'll want a colon here.

Commit:
Update Readme incorporating PR feedback

hassox commented:
:metal: Thanks dude. Updated.

bsidoruk committed:
changes for guardian 1.0

Commit:
Merge pull request #56 from aai/guardian_1_0

changes for guardian 1.0
Thanks @bsidoruk

doomspork commented on mix.exs:
```diff
@@ -28,10 +28,10 @@ defmodule GuardianDb.Mixfile do
 
   defp deps do
-    [{:guardian, "~> 0.14"},
+    [{:guardian, "~> 0.14 or >= 1.0"},
```
Do we need to change this to just `1.0`?

hassox committed:
Update README to 1.0 only. Bump version

cdesch committed:
change guardian 1.0  to 1.0.0-beta.0 

change guardian version reference from 1.0  to 1.0.0-beta.0

Commit:
updated postgrex to only >= 0.13

Hello,
I am getting below error when doing mix compile
** (Mix) Invalid requirement >= 0.11.1 or >= 0.13 for app postgrex
I think we either want one of >= 0.11.1 or >= 0.13 to fix this one.

Commit:
Merge pull request #59 from blisscs/patch-1

updated postgrex to only >= 0.13

LGTM

Commit:
Merge pull request #58 from kickinespresso/update-docs-for-guardian-1.0

change guardian 1.0  to 1.0.0-beta.0

Thanks :D

hassox commented:
@doomspork any other changes needed on this one?

doomspork reviewed (approved):
You're good to go @hassox 👍
